test/built-ins/object/values/exception-during-enumeration.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2015 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Object.values should terminate if getting a value throws an exception
+es7id: pending
+includes: [Test262Error.js]
+author: Jordan Harband
+---*/
+
+var trappedKey = {
+    get a() {
+        throw new Test262Error('This error should be re-thrown');
```

Comment:
`$ERROR` also throws a Test262Error, so you want to throw something other than a Test262Error.